app/scenes/load_team/load_team.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+
+import Loading from 'app/components/loading';
+
+import {RequestStatus} from 'service/constants';
+
+export default class LoadTeam extends React.Component {
+    static propTypes = {
+        teams: React.PropTypes.object.isRequired,
+        myMembers: React.PropTypes.object.isRequired,
+        teamsRequest: React.PropTypes.object.isRequired,
+        currentTeam: React.PropTypes.object,
+        actions: React.PropTypes.shape({
+            goToChannelView: React.PropTypes.func.isRequired,
+            handleTeamChange: React.PropTypes.func.isRequired,
+            initWebsocket: React.PropTypes.func.isRequired
+        }).isRequired
+    };
+
+    componentWillMount() {
+        this.props.actions.initWebsocket();
```

Comment:
Weren't we doing something different with the WebSocket connection or should we manage that in another PR?

My worry is that if the app is for a time in the background the websocket will disconnect, how is that going to be handled when the app goes back to the foreground.